src/main/groovy/com/google/protobuf/gradle/Utils.groovy

```diff
@@ -63,7 +63,7 @@ class Utils {
 
   static void addFilesToTaskInputs(Project project, TaskInputs inputs, Object files) {
     if (compareGradleVersion(project, "3.0") >= 0) {
-      inputs.file(files).skipWhenEmpty()
+      inputs.files(files).skipWhenEmpty()
```

Comment:
Wow, glad to see it can be fixed in such a clean way.

Reply:
Had to dig as deep as this diff to solve my Value `org.gradle.api.internal.tasks.DefaultTaskOutputs@1cc16f4b' specified for property '$1' cannot be converted to a file`. Thanks a lot!

Also, if you know how to debug such a problem (or how did you do it), please tell me.